tests/test_bootstrap.py

```diff
@@ -1146,6 +1146,7 @@ async def test_bootstrap_empty_integrations(
 @pytest.fixture(name="mock_mqtt_config_flow")
 def mock_mqtt_config_flow_fixture() -> Generator[None, None, None]:
     """Mock MQTT config flow."""
+    original_mqtt = HANDLERS.get("mqtt")
```

Comment:
Why aren't we using our test helper?

https://github.com/home-assistant/core/blob/67e199fb2f0c7940903176501a9ca5a07f88507a/tests/common.py#L1420